spec/support/csvs/spirits_for_import.csv

```diff
@@ -0,0 +1,9 @@
+Registered product name,Status,Class or category of product,Protection type,Country of origin,Traditional term grapevine product category,Traditional term type,Traditional term language,Date of application,Date of UK registration,Date of original registration with the EU,Decision notice,Protection instrument,Date of publication of the instrument,Legislation,Summary,Product type,Internal notes,,,,,,,,
```

Comment:
I think the convention is for these CSV files to be in `spec/fixtures`.

Reply:
That was my first call but then I seen in the code that this is not correct. Currently, other files like this used in similar integration tests live in this folder (for example: https://github.com/alphagov/specialist-publisher/tree/master/spec/support/images ) while the `spec/fixtures` contains factories https://github.com/alphagov/specialist-publisher/tree/master/spec/fixtures

Reply:
Hmm you're right, that's unfortunately. Okay let's stick with this for now.